.gitignore

```diff
@@ -19,3 +19,4 @@
 /app/javascript/test/unit/coverage/*
 /yarn.lock
 dump.rdb
+package-lock.json
```

Comment:
Are we ignoring `package-lock.json` now? In general, I thought it was good practice to check that in to control what versions were running in production.

Reply:
I think that it causes more problems than it's worth. It's the source of constant conflicts that can't be manually resolved.